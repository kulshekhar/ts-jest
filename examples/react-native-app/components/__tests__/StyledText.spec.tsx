import React from 'react'
import renderer from 'react-test-renderer'

import { MonoText } from '../StyledText'

it(`renders correctly`, () => {
  const tree = renderer.create(<MonoText>Snapshot test!</MonoText>).toJSON()

  expect(tree).toMatchSnapshot()
})
