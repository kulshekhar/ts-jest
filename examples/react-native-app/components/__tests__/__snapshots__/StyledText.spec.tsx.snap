// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000",
      },
      Array [
        undefined,
        Object {
          "fontFamily": "space-mono",
        },
      ],
    ]
  }
>
  Snapshot test!
</Text>
`;
